templates/cloudformation-template.yaml

```diff
@@ -89,11 +89,11 @@ Parameters:
 
   Domain:
     Type: String
-    Default: ''
+    Default: 'example.org'
 
   Email:
     Type: String
-    Default: ''
+    Default: 'test@example.org'
   AccessKey:
```

Comment:
can these be blank by default and dynamically required based on another flag HTTPSIsEnabled in the template (disabled by default)?

Reply:
also, these need to be applied to the other templates as well